### PR TITLE
Use editable panel component to edit application info

### PR DESCRIPTION
**Pull request by alexmt**

### CODE REVIEW

**Comment by AdityaGupta1** [comment]
I tried adding some info and saving it but all the info disappeared upon clicking save. Also, trying to use the other editable panel in the page (the one at the top) gives the following error:

Screen Shot 2019-06-12 at 2 57 28 PM

File: ui/src/app/applications/components/application-summary/application-summary.tsx [hunk 1]
```diff
@@ -108,158 +111,42 @@ export const ApplicationSummary = (props: {
...
+        view: info.value.match(urlPattern) ? <a href={info.value} target='__blank'>{info.value}</a> : info.value,
+        titleEdit: (formApi: FormApi) => <FormField formApi={formApi} field={`spec.info[${[i]}].name`} component={Text} componentProps={{style: { width: '99%' }}} />,
+        edit: (formApi: FormApi) => (
+            <React.Fragment>
+                <FormField formApi={formApi} field={`spec.info[${[i]}].value`} component={Text}/>
+                <i className='fa fa-times application-summary__remove-icon' onClick={() => {
+                    const values = (formApi.getFormState().values.spec.info || []) as Array<any>;
+                    formApi.setValue('spec.info', [...values.slice(0, i), ...values.slice(i + 1, values.length)]);
+                    setAdjustedCount(adjustedCount - 1);
+                }} style={{cursor: 'pointer'}}/>
```
Comment (AdityaGupta1) [inline ui/src/app/applications/components/application-summary/application-summary.tsx]: don't need inline styling here

**Comment by AdityaGupta1** [comment]
Also, using `EditablePanel` removes the ability to reorder the info.

File: ui/src/app/applications/components/application-summary/application-summary.tsx (hunk 1 above)
Comment (alexmt) [reply ui/src/app/applications/components/application-summary/application-summary.tsx]: done

**Comment by alexmt** [comment]
Thank you for catching JS error - fixed. Restored ability to reorder info.

**Pull request merged**